polaris-react/rollup.config.mjs

```diff
@@ -54,7 +54,7 @@ function generateConfig({output, targets, stylesConfig}) {
 /** @type {import('rollup').RollupOptions} */
 export default [
   generateConfig({
-    targets: 'extends @shopify/browserslist-config, node 16.13.0',
```

Comment:
Removing this seemed to be fine as it still builds with the config from `package.json`